### PR TITLE
updated thumbnail for wallpaper

### PR DESCRIPTION
**Pull request by elioqoshi**
updated the wallpaper thumbnail from Lunar to Mantic

### CODE REVIEW

File: templates/resources/index.html [hunk 1]
```diff
@@ -121,7 +121,7 @@ <h3 class="p-heading--5">Wallpapers</h3>
           class="col-4 u-hide--medium u-hide--small"
         >
           {{ image (
-            url="https://assets.ubuntu.com/v1/5d68f7aa-ubuntu-lunar-wallpaper-thumbnail.png",
+            url="https://assets.ubuntu.com/v1/ec6cc515-wallpaper_minotaur_colour_1920x1080.png",
             alt="",
             width="1500",
```
Comment (bartaz) [inline templates/resources/index.html:126]: @elioqoshi just to make sure image displays in correct aspect ratio please update the width and height props as well to be the same as image size (1920x1080).

Comment (elioqoshi) [reply templates/resources/index.html:126]: The aspect ratio should be the same, but I can update it as well.

**Comment by elioqoshi** [comment]
@bartaz is this good to go?

**Comment by bartaz** [comment]
> @bartaz is this good to go?

I think so, I'm just trying to get the demo running just to QA it properly.

**Review by bartaz — APPROVED** [review]

**Pull request merged**